README.md

```diff
@@ -30,21 +30,23 @@ These include: `Version`, `EventId`, `Timestamp`, `TenantId`, `EventBody.Timesta
 
 ### Setup
 
-Install from Bower:
+Either use `d2l-telemetry-browser-client` from the CDN or import it as an es6 module (assuming your application will transpile it for browsers that don't support es6 classes)
 
-```shell
-bower install Brightspace/d2l-telemetry-browser-client
-```
-
-Then import `d2l-telemetry-browser-client.html`:
+Reference the script in your html:
 
 ```html
-<link rel="import" href="../d2l-telemetry-browser-client/d2l-telemetry-browser-client.html">
+<script type="module" src="../d2l-telemetry-browser-client/d2l-telemetry-browser-client.js"></script>
 ```
 
-The HTML import will reference the library from the CDN, but using bower and html imports will allow you to de-dupe different versions.
+This will add a `d2lTelemetryBrowserClient` object to the global scope.
+
+Alternatively, you can reference the global d2lTelemetryBrowserClient instance via es6 import
+
+```javascript
+import { d2lTelemetryBrowserClient } from '../d2l-telemetry-browser-client/src/index.js';
```

Comment:
I think if you have a `module` entry in `package.json`, people can import it with `import d2lTelemetryBrowserClient from 'd2l-telemetry-browser-client';`, which is a little cleaner than needing to reference `index.js` directly IMO.

Example in `d2l-intl`:
https://github.com/Brightspace/intl/blob/master/package.json#L6

Reply:
Yeh, I think we have a couple of things going on here. We're trying to provide a mechanism that will provide out of the box backwards compatibility with existing usage and will work with how the polymer modulizer, converts references to the old HTML import to an es6 import.

From what I've seen, it will convert an import like:
    <link rel="import" href="../d2l-telemetry-browser-client/d2l-telemetry-browser-client.html">

to:
    import 'd2l-telemetry-browser-client/d2l-telemetry-browser-client.js';

So that was the reason for suggesting we add a new `d2l-telemetry-browser-client.js` file in the root that adds all the classes to the global scope.

But I think that is just for backwards compatibility.  Ideally we want to be steering people away from adding stuff to the window object and just importing and uses the classes directly.

So, I'm not even sure we need to show the backwards compatible solution as an example in the README.

If we want to provide a wrapper object like we do now as a default export (which is what d2l-intl is doing), we might need to provide a different `src/index.js` file, as the current one has references like:

     import {Client} from './client';
     module.exports.Client = Client;

Not sure if that format was required for the browserify build that we are using to publish to the CDN. But maybe with hindisight we should just ditch the browserify build and stop publishing to the CDN.

Then we can modify `src/index.js` to just create the default wrapper es6 export similar to what `d2l-intl/src/index.js` is doing.

Then modify the `package.json` `module` entry as Dave suggested, and the documented/recommended usage in the README will just be:

     import d2lTelemetryBrowserClient from 'd2l-telemetry-browser-client`;

And that will give you the wrapper object with all the classes as properties similar to what you would currrently get as a window global, but without the window global.

Reply:
Plus, if you're doing a major version bump and you want to encourage people to stop using the global you could just ditch it. They can stay on the old version for a while until they're ready to upgrade.